constrain/library/G36OutdoorAirDamperPositionForReturnFanAirflowTracking.py

```diff
@@ -1,34 +1,63 @@
 """
-G36 2021
-
 ### Description
 
 Section 5.16.2.3
+- Supply air temperature shall be controlled to setpoint using a control loop whose output is mapped to sequence the heating coil (if applicable), outdoor air damper, return air damper, and cooling coil.
```

Comment:
?

Reply:
Based on our group discussion, we've decided to leave it as is.